package.json

```diff
@@ -8,7 +8,10 @@
     "@testing-library/user-event": "^12.8.3",
     "react": "^17.0.2",
     "react-dom": "^17.0.2",
+    "react-icons": "^4.3.1",
```

Comment:
- [ ] [OPTIONAL] It seems that you've used React router dom version five, and I would suggest you use version six for it is the updated version 
Check this documentation for more [information ](https://reactrouter.com/docs/en/v6/getting-started/overview)about it